src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static String GREET_STRING = "Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?";
+    public static String BYE_STRING = "Bye. Hope to see you again soon!";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Scanner s = new Scanner(System.in);
+        InputReader inputReader = new InputReader();
+        TaskList taskList = new TaskList();
+
+        greet();
+
+        String input = s.nextLine();
+
+        while (true) {
+            if (input.equals("bye")) {
+                break;
+            } else if (input.equals("list")) {
+                taskList.displayTasks();
+            } else if (input.startsWith("unmark")) {
+                int index = inputReader.extractTaskIndexNo(input);
+                taskList.setTaskStatus(index, false);
+            } else if (input.startsWith("mark")) {
+                int index = inputReader.extractTaskIndexNo(input);
+                taskList.setTaskStatus(index, true);
+            } else if (input.startsWith("todo")) {
+                String toDoTask = inputReader.extractToDoTask(input);
+                taskList.addToDo(toDoTask);
+            } else if (input.startsWith("deadline")) {
+                String[] deadlineTask = inputReader.extractDeadlineTask(input);
+                taskList.addDeadline(deadlineTask[0], deadlineTask[1]);
+            } else if (input.startsWith("event")) {
+                String[] eventTask = inputReader.extractEventTask(input);
+                taskList.addEvent(eventTask[0], eventTask[1]);
+            } else {
+                promptAgain();
+            }
+            input = s.nextLine();
+        }
+        bye();
+
     }
+
+    private static void promptAgain() {
+        System.out.println("Try again!");
```

Comment:
Would it be more consistent to use a `String` constant like in the two methods below?

Reply:
Will be implementing this together with `Level-5`